src/displays/menus/controllerhardwaresettingsmenu.h

```diff
@@ -23,13 +23,14 @@ class SettingsMenu;
 }
 
 namespace {
-using WheelDiameterMmChangeScreen = makeComponent<ChangeValueDisplay<int16_t>, StaticText<TEXT_WHEELDIAMETERMM>, WheelDiameterMmAccessor, SwitchScreenAction<ControllerHardwareSettingsMenu>>;
-using WheelDiameterInchChangeScreen = makeComponent<ChangeValueDisplay<float>, StaticText<TEXT_WHEELDIAMETERINCH>, WheelDiameterInchAccessor, RatioNumberStep<float, std::ratio<1,10>>, SwitchScreenAction<ControllerHardwareSettingsMenu>>;
-using NumMagnetPolesChangeScreen = makeComponent<ChangeValueDisplay<int16_t>, StaticText<TEXT_NUMMAGNETPOLES>, NumMagnetPolesAccessor, SwitchScreenAction<ControllerHardwareSettingsMenu>>;
+using WheelDiameterMmChangeScreen = makeComponent<ChangeValueDisplay<int16_t>, StaticText<TEXT_WHEELDIAMETERMM>, WheelDiameterMmAccessor, BackActionInterface<SwitchScreenAction<ControllerHardwareSettingsMenu>>, SwitchScreenAction<ControllerHardwareSettingsMenu>>;
+using WheelDiameterInchChangeScreen = makeComponent<ChangeValueDisplay<float>, StaticText<TEXT_WHEELDIAMETERINCH>, WheelDiameterInchAccessor, RatioNumberStep<float, std::ratio<1,10>>, BackActionInterface<SwitchScreenAction<ControllerHardwareSettingsMenu>>, SwitchScreenAction<ControllerHardwareSettingsMenu>>;
+using NumMagnetPolesChangeScreen = makeComponent<ChangeValueDisplay<int16_t>, StaticText<TEXT_NUMMAGNETPOLES>, NumMagnetPolesAccessor, BackActionInterface<SwitchScreenAction<ControllerHardwareSettingsMenu>>, SwitchScreenAction<ControllerHardwareSettingsMenu>>;
```

Comment:
Maybe do something like this to make these lines a bit shorter.
```suggestion
using NumMagnetPolesChangeScreen = makeComponent<
        ChangeValueDisplay<int16_t>,
        StaticText<TEXT_NUMMAGNETPOLES>,
        NumMagnetPolesAccessor,
        BackActionInterface<SwitchScreenAction<ControllerHardwareSettingsMenu>>,
        SwitchScreenAction<ControllerHardwareSettingsMenu>
    >;
```